### PR TITLE
LPM-5099 Modified mulesoft user credentials to use connected app

### PR DESCRIPTION
**Pull request by Muhammad-Sarmad-khattak**
jira: LPM-5099

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -18,8 +18,8 @@ async function main() {
   const MULESOFT_NEXUS_USER = process.env.MULESOFT_NEXUS_USER;
   const MULESOFT_NEXUS_PASSWORD = process.env.MULESOFT_NEXUS_PASSWORD;
   const PAGERDUTY_INTEGRATION_KEY = process.env.PAGERDUTY_INTEGRATION_KEY;
-  const MULESOFT_DEPLOY_USERNAME = process.env.MULESOFT_DEPLOY_USERNAME;
-  const MULESOFT_DEPLOY_PASSWORD = process.env.MULESOFT_DEPLOY_PASSWORD;
+  const MULESOFT_CONNECT_APP_ID = process.env.MULESOFT_CONNECT_APP_ID;
```
Comment (xavier07csm) [inline src/index.js:21]:
```suggestion
  const MULESOFT_CONNECTED_APP_ID = process.env.MULESOFT_CONNECTED_APP_ID;
```

File: src/index.js [hunk 2]
```diff
@@ -18,8 +18,8 @@ async function main() {
...
+  const MULESOFT_CONNECT_APP_SECRET = process.env.MULESOFT_CONNECT_APP_SECRET;
```
Comment (xavier07csm) [inline src/index.js:22]:
```suggestion
  const MULESOFT_CONNECTED_APP_SECRET = process.env.MULESOFT_CONNECTED_APP_SECRET;
```

File: src/index.js [hunk 3]
```diff
@@ -33,7 +33,7 @@ async function main() {
       }
       return;
     }
-    if (await maven.build(SECRET_KEY, MULESOFT_NEXUS_USER, MULESOFT_NEXUS_PASSWORD, MULESOFT_DEPLOY_USERNAME , MULESOFT_DEPLOY_PASSWORD)) {
+    if (await maven.build(SECRET_KEY, MULESOFT_NEXUS_USER, MULESOFT_NEXUS_PASSWORD, MULESOFT_CONNECT_APP_ID , MULESOFT_CONNECT_APP_SECRET)) {
```
Comment (xavier07csm) [inline src/index.js:36]:
```suggestion
    if (await maven.build(SECRET_KEY, MULESOFT_NEXUS_USER, MULESOFT_NEXUS_PASSWORD, MULESOFT_CONNECTED_APP_ID , MULESOFT_CONNECTED_APP_SECRET)) {
```

**Review by xavier07csm — CHANGES_REQUESTED** [review]
The secrets are MULESOFT_DEPLOY_CONNECTED_APP_*  So let's keep the naming the same for the inputs.

**Review by xavier07csm — APPROVED** [review]

**Review by jamestflynn — APPROVED** [review]

**Pull request merged**